test/parallel/test-https-set-timeout-server.js

```diff
@@ -34,8 +35,8 @@ const tls = require('tls');
 const tests = [];
 
 const serverOptions = {
-  key: fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/keys/agent1-cert.pem`)
+  key: fs.readFileSync(`${fixtures.fixturesDir}/keys/agent1-key.pem`),
+  cert: fs.readFileSync(`${fixtures.fixturesDir}/keys/agent1-cert.pem`)
```

Comment:
There is a dedicated `fixtures.readKey` function that should be used here. In that case the line is much shorter and you only have to provide the file name.

Reply:
@BridgeAR, so for example?

```
key: fixtures.readKey(`agent1-key.pem`),
```